samples/openthread/cli/README.rst

```diff
@@ -45,7 +45,7 @@ Experimental Thread 1.2 extension
 =================================
 
 This optional extension allows you to test :ref:`available features from the Thread 1.2 Specification <thread_ug_thread_1_2>`.
-You can enable these features either by :ref:`activating the overlay extension <ot_cli_sample_activating_variants>` as described below or by setting :ref:`thread_ug_thread_1_2`.
+You can enable these features by setting :ref:`thread_ug_thread_1_2`.
```

Comment:
Activating extra Thread v1.2 features by cli sample are hidden for end user right now. Before your change these features where listed in cli overlay-trhead_1_2.conf file, now they are implicitly enabled with this option: CONFIG_OPENTHREAD_NORDIC_LIBRARY_MASTER=y in cli prj.conf file. This is not obvious what is happening at the first glance. I think we need to change it by at least better documenting.